.ci-builds/.azure-pipelines.yml

```diff
@@ -582,3 +582,44 @@ jobs:
       env:
         CC: gcc
         CXX: g++
+#-------------------------------------------------------------------------
+- job: ICU4C_icuwriteuprops_Ubuntu_1804
+  displayName: 'C: Create Artifacts from icuwriteuprops'
+  timeoutInMinutes: 30
+  pool:
+    vmImage: 'ubuntu-18.04'
+  steps:
+    - checkout: self
+      lfs: true
+      fetchDepth: 10
+    - script: |
+        cd icu4c/source && ./runConfigureICU Linux && make -j2
+      displayName: 'Build'
+      env:
+        CC: clang
+        CXX: clang++
+    - script: |
+        mkdir icu4c/source/full_out
+        cd icu4c/source && LD_LIBRARY_PATH=lib ./bin/icuwriteuprops -A -I -c -d full_out
+        zip -rj icuwriteuprops_full.zip full_out/
+      displayName: 'Build full property dump file'
+    # In the sample file, include:
+    # - Basic binary properties: ASCII_Hex_Digit Basic_Emoji White_Space
+    # - Property of strings: RGI_Emoji
```

Comment:
Basic_Emoji also has strings. Therefore, you may want to not also add RGI_Emoji to samples.